build-support/bin/native/bootstrap.sh

```diff
@@ -60,6 +60,7 @@ function calculate_current_hash() {
      "${REPO_ROOT}/src/python/pants/engine/native.py" \
      "${REPO_ROOT}/build-support/bin/native" \
      "${REPO_ROOT}/3rdparty/python/requirements.txt" \
+   | grep -v -e "/BUILD" -e "/*.md" \
```

Comment:
Probably doesn't matter, but neither of these are anchored. Should probably be `/BUILD$`, etc?

Reply:
I think if I throw in a -E that should work on both BSD and GNU grep.